wurst/objects/items/BootsDefinition.wurst

```diff
@@ -0,0 +1,244 @@
+                                    getAbilityDataField(ABILITY_ARMOR_BONUS_5,   "DataA1").color(COLOR_GREEN),
+                                    5..toToolTipGreen(),
+                                    getAbilityDataField(ABILITY_SPELL_SHIELD, "Cool1").color(COLOR_LIGHT_BLUE)
+                                    )
+
+@compiletime function createSpeedBonusAbility()
+    new AbilityDefinitionMoveSpeedBonus(ABILITY_SPEED_BONUS_60)
+        ..setMovementSpeedBonus(1, 60)
+        ..setName("Movement Speed Bonus 60")
+        ..setEditorSuffix("(Wurst)")
+
+    new AbilityDefinitionMoveSpeedBonus(ABILITY_SPEED_BONUS_70)
+        ..setMovementSpeedBonus(1, 70)
+        ..setName("Movement Speed Bonus 70")
+        ..setEditorSuffix("(Wurst)")
+
+    new AbilityDefinitionMoveSpeedBonus(ABILITY_SPEED_BONUS_80)
+        ..setMovementSpeedBonus(1, 80)
+        ..setName("Movement Speed Bonus 80")
+        ..setEditorSuffix("(Wurst)")
+
+    new AbilityDefinitionMoveSpeedBonus(ABILITY_SPEED_BONUS_100)
+        ..setMovementSpeedBonus(1, 100)
+        ..setName("Movement Speed Bonus 100")
+        ..setEditorSuffix("(Wurst)")
+
+    new AbilityDefinitionMoveSpeedBonus(ABILITY_SPEED_BONUS_200)
+        ..setMovementSpeedBonus(1, 200)
+        ..setArtTarget(Abilities.faerieDragonMissile)
+        ..setTargetAttachmentPoint("foot,left")
+        ..setTargetAttachmentPoint1("foot,right")
+        ..setTargetAttachments(2)
+        ..setName("Movement Speed Bonus 200")
+        ..setEditorSuffix("(Wurst)")
+
+@compiletime function createEvasionAbility()
+    new AbilityDefinitionEvasion(ABILITY_ANABOLIC_EVASION)
+        ..setChancetoEvade(1, 0.10)
+        ..setName("Anabolic Evasion")
+        ..setEditorSuffix("(Wurst)")
+
+@compiletime function createSpellShieldAbility()
+    new AbilityDefinition(ABILITY_SPELL_SHIELD, 'ANss')
+        ..setCooldown(1, 75.0)
+        ..setName("Hydra Fins Spell Shield (Amulet)")
+        ..setEditorSuffix("(Wurst)")
+
+@compiletime function createElkSkinBoots() returns ItemDefinition
+    return createBasicItem(ITEM_ELK_SKIN_BOOTS)
+        ..setAbilities(commaList(ABILITY_SPEED_BONUS_60, ABILITY_INT_BONUS_2))
+        ..setInterfaceIcon(LocalIcons.bTNElkHideBoots)
+        ..setDescription("Increases wearer's movement speed and intelligence")
+        ..setTintingColor1Red(255)
+        ..setTintingColor2Green(255)
+        ..setTintingColor3Blue(255)
+        ..setModelUsed(LocalItems.skinBoots)
+        ..setHotkey("")
+        ..setLumberCost(19)
+        ..setNameEnhance("Elk Skin Boots")
+        ..setTooltipExtended(TT_ELK_SKIN_BOOTS)
+        ..setCooldownGroup(toRawCode(ABILITY_SPEED_BONUS_60))
+
+@compiletime function createWolfSkinBoots() returns ItemDefinition
+    return createBasicItem(ITEM_WOLF_SKIN_BOOTS)
+        ..setAbilities(commaList(ABILITY_SPEED_BONUS_60, ABILITY_INT_BONUS_4, ABILITY_ARMOR_BONUS_2))
```

Comment:
I think in general we should use more lines for nested calls, but this is fine right now. Just more likely to change if more abilities are added.